book/website/afterword/glossary.md

```diff
@@ -405,8 +405,8 @@ Open Scholarship
 Open Project
   *Same as Open Science or Open Research Projects*. A project in which a significant amount of collaboration between the core or leadership team and the wider community takes place in the form of online interactions. Community interactions should maintain transparency and openness of the project to facilitate the growth of your community.
 
-Open Source Software
-  Documenting research code and routines, and making them freely accessible and available.
+Open Source
+  Open source software is software whose source code can be freely used, modified, and shared (in both modified and unmodified form). Today the concept of "open source" is often extended beyond software, to represent a philosophy of collaboration in which source files of digital resources are made available online to be used, modified and shared. (Ex: open source resource, open source educational resource, open source image, etc.)
```

Comment:
```suggestion
  Often refers specifically to open source software.
  Today the concept is often extended beyond software, to represent a philosophy of collaboration in which source files of digital resources are made freely available to be used, modified and shared.
  Examples of open source which are not specifically software include open research, open data, open educational resources and open hardware.
```

Some tweaks to change how this reads.
Also trying to avoid Latin and abbreviations as per the style guide.

Reply:
I switched to a multiline definition. I updated the examples sentence with the following, let me know what you think of that:
> Examples of open source artefacts which are not specifically software encompass open source resources, open source educational resources, open source hardware or open source image.

I think this aspect of « open source resource » and derivates are important to explicit the concept, being something different than open research(/science), open data, open education, open hardware & other open models. Open source is related to the digital resources manipulated in these different domains, open source resources are what may be used in open research for example.

There is sometimes a tendency to assimilate open source to open science/education, like it would be open source outside of software (the unclear "principles") despite the important subtlety between these notions. A possible source of confusion.

![difference-open-source-open-models](https://github.com/the-turing-way/the-turing-way/assets/34010605/38e777d2-5621-4195-898d-d1617496997f)